components/helpers/SiteMessages.php

```diff
@@ -0,0 +1,33 @@
+<?php
+//******************************************************************************
+//                                       SiteMessages.php
+// SILEX-PHIS
+// Copyright © INRA 2018
+// Creation date: 10 Sept, 2018
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\components\helpers;
```

Comment:
Do plan to make it as a helper ?
We need to change everywhere, maybe it could have been a specific branch...